src/Domain/ElementDistribution.hpp

```diff
@@ -65,6 +64,15 @@ namespace domain {
  * globally, though it would likely be more efficient to prioritize minimization
  * of inter-node communication, because communication across interconnects is
  * the primary cost of communication in charm++ runs.
+ *
+ * \warning The use of the Morton curve to generate a well-clustered element
+ * distribution currently assumes that the refinement is uniform over each
+ * block, with no internal structure that would be generated by, for instance
+ * AMR.
+ * This distribution method will need alteration to perform well for blocks with
+ * internal refinement structure. Morton curves can be defined recursively, so a
+ * generalization of the present method is possible for blocks with internal
```

Comment:
by "internal refinement" do you mean `h` or `p` or both? I think that would be good to clarify